pkg/azure/arm/deployer.go

```diff
@@ -36,6 +36,15 @@ type Deployer interface {
 		armParams map[string]interface{},
 		tags map[string]string,
 	) (map[string]interface{}, error)
+	Update(
+		deploymentName string,
```

Comment:
i rebased off my other open PR and this came with it, so review this in the other one.